package.json

```diff
@@ -36,34 +37,64 @@
     "fix": "npm run fix:lint && npm run fix:prettier",
     "start": "npm run develop",
     "serve": "gatsby serve",
-    "test": "echo \"Write tests! -> https://gatsby.app/unit-testing\"",
     "commit": "git-cz",
-    "storybook": "start-storybook"
+    "storybook": "start-storybook",
+    "cypress:open": "cross-env TZ=UTC cypress open",
+    "cypress:run": "cross-env TZ=UTC cypress run",
+    "test": "nyc --silent cross-env npm run test:unit && npm run test:e2e",
+    "test:unit": "jest",
+    "test:e2e": "start-server-and-test develop http://localhost:8000 cypress:run"
   },
   "devDependencies": {
     "@storybook/addon-docs": "^5.2.8",
     "@storybook/addon-viewport": "^5.2.8",
     "@storybook/react": "5.2.8",
+    "@cypress/code-coverage": "^1.10.2",
     "@typescript-eslint/eslint-plugin": "2.7.0",
     "@typescript-eslint/parser": "2.7.0",
     "babel-eslint": "10.0.3",
     "babel-loader": "^8.0.6",
     "commitizen": "^4.0.3",
-    "cssnano": "^4.1.10",
     "cz-conventional-changelog": "^3.0.2",
+    "babel-jest": "^24.9.0",
+    "babel-plugin-istanbul": "^5.2.0",
+    "babel-preset-gatsby": "^0.2.23",
+    "chai": "^4.2.0",
+    "chai-enzyme": "^1.0.0-beta.1",
+    "coveralls": "^3.0.9",
+    "cross-env": "^6.0.3",
+    "cssnano": "^4.1.10",
+    "cypress": "^3.7.0",
+    "dotenv": "8.2.0",
```

Comment:
This dependency is defined twice in the `dependency` and `devDependency` of the `package.json`, I think it's safe to remove one of them.